OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/AdminControllerTests.cs

```diff
@@ -0,0 +1,780 @@
+            Institution = w.Institution,
+            AvailableSeats = w.AvailableSeats ?? uint.MaxValue,
+            TakenSeats = w.TakenSeats,
+        }).ToList();
+    }
+
+    private WorkshopDescriptionItemDto FakeWorkshopDescriptionItem()
+    {
+        var id = Guid.NewGuid();
+        return new WorkshopDescriptionItemDto
+        {
+            Id = id,
+            SectionName = "test heading",
+            Description = $"test description text sentence for id: {id.ToString()}",
+        };
+    }
+
+    private WorkshopV2Dto FakeWorkshop()
+    {
+        return new WorkshopV2Dto()
+        {
+            Id = Guid.NewGuid(),
+            Title = "Title6",
+            Phone = "1111111111",
+            WorkshopDescriptionItems = new[]
+            {
+                FakeWorkshopDescriptionItem(),
+                FakeWorkshopDescriptionItem(),
+                FakeWorkshopDescriptionItem(),
+            },
+            Price = 6000,
+            WithDisabilityOptions = true,
+            ProviderTitle = "ProviderTitle",
+            DisabilityOptionsDesc = "Desc6",
+            Website = "website6",
+            Instagram = "insta6",
+            Facebook = "facebook6",
+            Email = "email6@gmail.com",
+            MaxAge = 10,
+            MinAge = 4,
+            CoverImageId = "image6",
+            ProviderId = Guid.NewGuid(),
+            InstitutionHierarchyId = new Guid("af475193-6a1e-4a75-9ba3-439c4300f771"),
+            AddressId = 55,
+            Address = new AddressDto
+            {
+                Id = 55,
+                CATOTTGId = 4970,
+                Street = "Street55",
+                BuildingNumber = "BuildingNumber55",
+                Latitude = 0,
+                Longitude = 0,
+            },
+            Teachers = new List<TeacherDTO>
+            {
+                new TeacherDTO
+                {
+                    Id = Guid.NewGuid(),
+                    FirstName = "Alex",
+                    LastName = "Brown",
+                    MiddleName = "SomeMiddleName",
+                    Description = "Description",
+                    CoverImageId = "Image",
+                    DateOfBirth = DateTime.Parse("2000-01-01"),
+                    WorkshopId = new Guid("5e519d63-0cdd-48a8-81da-6365aa5ad8c3"),
+                },
+                new TeacherDTO
+                {
+                    Id = Guid.NewGuid(),
+                    FirstName = "John",
+                    LastName = "Snow",
+                    MiddleName = "SomeMiddleName",
+                    Description = "Description",
+                    CoverImageId = "Image",
+                    DateOfBirth = DateTime.Parse("1990-01-01"),
+                    WorkshopId = new Guid("5e519d63-0cdd-48a8-81da-6365aa5ad8c3"),
+                },
+            },
+        };
+    }
+    private List<WorkshopV2Dto> FakeWorkshops()
+    {
+        return new List<WorkshopV2Dto>()
+        {
+            new WorkshopV2Dto()
```

Comment:
(Test helpers and code reuse) Please use existing (if any) or write new faker to generate `WorkshopV2Dto`s.

Reply:
You've changed nothing here and not responded. But set the comment to resolved. I'm sad